step-4/build.gradle

```diff
@@ -0,0 +1,80 @@
+buildscript {
```

Comment:
You could be using the simpler `plugins { ... }` block since there are no special dependencies.